### DISCUSSION TITLE
Desired Stance won't update Stance

### DISCUSSION THREAD

**Discussion by sommerlost** [Sixze/ALS-Refactored#243] [Q&A] [answered]
I am having a hell of a time trying to update the Stance based upon a new tag. Of course setting the desired tag to Standing or Crouching works as expected, but it will not respect a new tag such as 'Als.Stance.Sitting'.

For sanity,

- I've updated MS_Normal
![image](https://user-images.githubusercontent.com/7546149/218846356-94141579-783d-4cee-9ce0-c0f99a87028f.png)

- Set Desired Stance is being called on both Server/Client

- I've even added everything to the anim graph just to be sure:
![image](https://user-images.githubusercontent.com/7546149/218847077-f0701c95-b603-41b0-94d2-ab72721969a5.png)

- I've dug through the code, and added an else if statement for my sitting tag and called a print message just to make sure things are connecting, which they are.
![image](https://user-images.githubusercontent.com/7546149/218853815-637b0384-09d2-4d2b-a936-a56365a52e05.png)
It looks like a tag might actually be called in the UnCrouch(); or Crouch(); events, but I still haven't found where this happens. Those just look like call to the base Character.cpp

I am a bit stuck here if anyone has any ideas what I'm missing.

**Comment by sommerlost** [comment, marked as answer]
I've learned if I want to figure something out, I have to just make a post then I'll finally stumble on the answer.

This may not be the best method but it works for me, I just added a few lines to AlsCharacter.cpp:
![image](https://user-images.githubusercontent.com/7546149/218856425-a8ecf15a-827c-44ca-9a39-a1e5997fd43b.png)

I actually ended up moving the line up a bit so I could change the stance back after I reverted to standing state:

![image](https://user-images.githubusercontent.com/7546149/218859257-f8dc4326-ecd2-475d-9b9a-949a7084ce5b.png)

**Comment by xkamronx** [comment]
Did you run in to any issues with anything else after adding that line to force the stance change?
